code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -231,24 +247,31 @@
 	if(location.excited_group)
 		location.excited_group.reset_cooldowns()
 
-	if((temperature < FIRE_MINIMUM_TEMPERATURE_TO_EXIST) || (volume <= 1))
+	cold_fire = FALSE
+	if(temperature <= FREON_MAXIMUM_BURN_TEMPERATURE)
+		cold_fire = TRUE
+
+	if((temperature < FIRE_MINIMUM_TEMPERATURE_TO_EXIST && !cold_fire) || (volume <= 1))
 		qdel(src)
 		return
 
 	//Not enough / nothing to burn
-	if(!location.air || (INSUFFICIENT(/datum/gas/plasma) && INSUFFICIENT(/datum/gas/tritium) && INSUFFICIENT(/datum/gas/hydrogen)) || INSUFFICIENT(/datum/gas/oxygen))
+	if(!location.air || (INSUFFICIENT(/datum/gas/plasma) && INSUFFICIENT(/datum/gas/tritium) && INSUFFICIENT(/datum/gas/hydrogen) && INSUFFICIENT(/datum/gas/freon)) || INSUFFICIENT(/datum/gas/oxygen))
 		qdel(src)
 		return
 
 	perform_exposure()
 
 	if(bypassing)
 		icon_state = "3"
-		location.burn_tile()
+		if(location.air.temperature > FIRE_MINIMUM_TEMPERATURE_TO_EXIST)
```

Comment:
```suggestion
		if(!cold_fire)
```
?

Reply:
this checks the location.air i don't know if that temperature is equal to the hotspot temperature, so i have no idea if is a good idea to swap them